pages/[slug].js

```diff
@@ -427,6 +427,19 @@ export default function Post({ postData, status, categoryPosts }) {
                         : ""}
                     </div>
                   </div>
+                  {/* Recommended Posts Section Desktop View */}
+
+                  {categoryPosts.length == 0 ? (
+                    ""
+                  ) : (
+                    <div className="realtive w-[30%]">
```

Comment:
Can't we make the div more wider so the title(`recommended for you`) will be in one line?